api/app/controllers/mno_enterprise/jpi/v1/admin/dashboard_templates_controller.rb

```diff
@@ -0,0 +1,90 @@
+module MnoEnterprise
+  class Jpi::V1::Admin::DashboardTemplatesController < Jpi::V1::Admin::BaseResourceController
+
+    #==================================================================
+    # Instance methods
+    #==================================================================
+    # GET /mnoe/jpi/v1/admin/dashboard_templates
+    def index
+      if params[:terms]
+        # For search mode
+        @dashboard_templates = []
```

Comment:
You should not user current_user.dashboard, but MnoEnterprise::Dashboard directly, and filter on the dashboard where dashboard_type = 'template'